### PR TITLE
Reworks Lichdom

### PR DESCRIPTION
**Pull request by Alberyk**
Instead of just turning the wizard into a skeleton with the resurrection proc, the lichdom spell will now create a phylactery, bound to the wizard. When the lich dies, he can use a skill to return to the phylactery, where he will come back after to life some time. If the phylactery is destroyed, the lich will be unable to return to life. Trying to use dark resurrection without a phylactery will result in the lich's body being dusted.

Also, adds a new sprite for the wizard book.

### CODE REVIEW

File: code/modules/spells/targeted/necromancer_spells.dm [hunk 1]
```diff
@@ -141,15 +141,63 @@
...
+			playsound(C.loc, 'sound/hallucinations/wail.ogg', 50, 1)
+			C.dust()
+			return
+
+		C.forceMove(get_turf(N))
+
+	C << "<span class='notice'>Your dead body returns to your phylactery, slowly rebuilding itself.</span>"
+	C.verbs -= /mob/living/carbon/proc/dark_resurrection
+
+	spawn(rand(400,800))
```
Comment (Lohikar) [inline code/modules/spells/targeted/necromancer_spells.dm]: Timer pls.

File: code/modules/spells/artifacts.dm [hunk 2]
```diff
@@ -119,3 +119,42 @@
...
+	
+	var/lich = null
+
+/obj/item/phylactery/New()
+	..()
+	world_phylactery += src
+
+/obj/item/phylactery/Destroy()
+	lich << "<span class='danger'>Your phylactery was destroyed, your soul is cast into the abyss as your immortality vanishes away!</span>"
+	world_phylactery -= src
```
Comment (Lohikar) [inline code/modules/spells/artifacts.dm]: Should probably null out `lich` here, just to be safe.

File: code/modules/spells/targeted/necromancer_spells.dm [hunk 3]
```diff
@@ -141,15 +141,62 @@
...
+		 
+	if(P)
+		C.forceMove(get_turf(P))
+		C << "<span class='notice'>Your dead body returns to your phylactery, slowly rebuilding itself.</span>"
+		C.verbs -= /mob/living/carbon/proc/dark_resurrection
+		addtimer(CALLBACK(src, .proc/post_dark_resurrection), rand(400, 800))
+
+	else
+		C << "<span class='danger'>Your phylactery was destroyed, your existence will face oblivion now.</span>"
+		C.visible_message("<span class='cult'>\The [C]'s body turns into dust, a twisted wail can be heard!</span>")
```
Comment (Lohikar) [inline code/modules/spells/targeted/necromancer_spells.dm]: The grammar here is a little dicey. Perhaps `As [C]'s body turns to dust, a twisted wail can be heard!`

File: code/modules/spells/artifacts.dm (hunk 2 above)
Comment (skull132) [inline code/modules/spells/artifacts.dm]: `lich = null` yesss

Comment (Alberyk) [inline code/modules/spells/artifacts.dm]: Done.

**Pull request merged**